dash.js

```diff
@@ -277,6 +278,12 @@ var MyDash = new Lang.Class({
 
         this._appSystem = Shell.AppSystem.get_default();
 
+        // Remove Drive Icons
+        this._removables = new Locations.Removables();
+
+        // Trash Icon
+        this._trash = new Locations.Trash();
```

Comment:
Would it be possible to create the `_trash` object only if enabled in the settings?

I actually disabled it a long time ago, and only today I realized why my system froze for a few seconds every time I deleted files: the call back to trash changes is still active!